samples/multiplatform/build.gradle

```diff
@@ -0,0 +1,42 @@
+plugins {
+    id 'org.jetbrains.kotlin.multiplatform' version '1.3.61'
+}
+repositories {
+    mavenCentral()
+}
+group 'com.example'
+version '0.0.1'
+
+apply plugin: 'maven-publish'
+
+kotlin {
+    jvm()
+    js {
+        browser {
+        }
+        nodejs {
+        }
+    }
+    // For ARM, should be changed to iosArm32 or iosArm64
+    // For Linux, should be changed to e.g. linuxX64
+    // For MacOS, should be changed to e.g. macosX64
+    // For Windows, should be changed to e.g. mingwX64
+    mingwX64("windows")
+    sourceSets {
+
```

Comment:
please also add the dependency for commonTest (and androidTest once you have setup the android target). Something along the line of
```
commonTest {
    dependencies {
        implementation kotlin('test-common')
        implementation kotlin('test-annotations-common')
        implementation kotlin('ch.tutteli.atrium:atrium-fluent-en_GB-common:0.9.0-alpha2')
    }
}
```